client/reader/discover/discover-stream.js

```diff
@@ -80,8 +82,16 @@ const DiscoverStream = ( props ) => {
 	);
 
 	const streamSidebar = () => {
-		if ( selectedTab === FIRST_POSTS_TAB ) {
-			return <></>;
+		if ( selectedTab === FIRST_POSTS_TAB && recommendedSites?.length ) {
+			return (
+				<>
+					<h2>{ translate( 'New Sites' ) }</h2>
```

Comment:
```suggestion
					<h2>{ translate( 'New sites' ) }</h2>
```

I think this should probably be "New sites" because we use sentence-case capitalization elsewhere in Calypso (e.g. "Manage subscribed sites").

Reply:
I wondered about that, too. We use "Popular Sites" in the "Recommended" tab sidebar, so if we change it here, we should probably change that one as well.

Reply:
This should be fixed now!